### PR TITLE
Datepicker component

### PR DESCRIPTION
**Pull request by ckoga**
### What's this PR do?
This PR adds a DatePicker component that allows a user to select a range of dates to be used to create a Trip.
### Where should the reviewer start?
To start take a look at the documentation for [react-native-calendars](https://github.com/wix/react-native-calendars) and [moment.js](https://momentjs.com/docs/).  We are using these dependencies for the Calendar component and methods for formatting dates.
### How should this be manually tested?
Pull down the datepicker-component branch and run npm i to install the new dependencies.
run npm start 
navigate to the calendar in the CreateTrip component and see if you can select a range of dates.  The dates should be highlighted when done correctly.
### What are the relevant tickets?
This PR addresses issue #7

### CODE REVIEW

**Comment by ckoga** [comment]
I put up both gifs to highlight styling not working the same between browser and virtual android.

File: Components/DatePicker/DatePicker.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            justifyContent: 'space-around'
+          }
+        }
+      }}
+      markedDates={markedDates}
+    />
+  )
+}
+
+export default DatePicker;
```
Comment (sertmer) [inline Components/DatePicker/DatePicker.js]: Damn! Nice work! I appreciate all that went into this - Setting up state and hooks, the logic for the behavior, and the styling!

Comment (sertmer) [reply Components/DatePicker/DatePicker.js]: Is there a way to select a month?

Comment (ckoga) [reply Components/DatePicker/DatePicker.js]: Yeah so one thing I decided to leave for later is making the arrows show up.  Yptehy are there and can be used to navigate months but no matter what css property I used/changed I couldn’t get them to show.  Another thing I just noticed is not matter what date you select the screen will snap back to the current date.  I’d like for that to not happen.

**Pull request merged**